PyOpenWorm/dataObject.py

```diff
@@ -548,14 +542,14 @@ def _create_property_class(
             c = PropertyTypes[_PropertyTypes_key]
         else:
             klass = None
-            if property_type == 'ObjectProperty':
+            if property_type == ObjectProperty.__name__:
                 if value_type is not None and value_rdf_type is None:
                     value_rdf_type = value_type.rdf_type
                 klass = SP.ObjectProperty
-            elif property_type == 'DatatypeProperty':
+            elif property_type == DatatypeProperty.__name__:
                 value_rdf_type = None
                 klass = SP.DatatypeProperty
-            elif property_type == 'UnionProperty':
+            elif property_type == UnionProperty.__name__:
```

Comment:
This usage of `__name__` is inappropriate. Yes, the property classmethods and the property descriptors (via `APThunk`) do pass in keys to this method, so it makes sense to use a constant that they share, but the UnionProperty/DatatypeProperty/ObjectProperty defined in the top level of this module shouldn't be dependencies of this code as that would be circular. If anything, you could define constants that they all share.